libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -1465,12 +1602,26 @@ protected void onAppBackgrounded() {
         ((ScreenLockManager) getScreenLockManager()).onAppBackgrounded();
         ((BiometricAuthenticationManager) getBiometricAuthenticationManager())
                 .onAppBackgrounded();
+
+        // Hide the Salesforce Mobile SDK "Show Developer Support" notification
+        if (authenticatedActivityForDeveloperSupport != null) {
+            ShowDeveloperSupportNotifier.Companion.hideDeveloperSupportNotification(
+                    authenticatedActivityForDeveloperSupport
+            );
+        }
     }
 
     @OnLifecycleEvent(Lifecycle.Event.ON_START)
     protected void onAppForegrounded() {
         ((ScreenLockManager) getScreenLockManager()).onAppForegrounded();
         ((BiometricAuthenticationManager) getBiometricAuthenticationManager())
                 .onAppForegrounded();
+
+        // Display the Salesforce Mobile SDK "Show Developer Support" notification
+        if (getUserAccountManager().getCurrentAccount() != null && authenticatedActivityForDeveloperSupport != null) {
```

Comment:
I piggy-backed on this existing application lifecycle event to show the notification when the app is delivered to the foreground.  This includes an extra check to ensure the notification isn't shown without authentication.